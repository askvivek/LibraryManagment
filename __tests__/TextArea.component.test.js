import React from 'react';
import renderer from 'react-test-renderer';
import TextArea from '../app/components/TextArea/TextArea.component';

describe('Some component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <TextArea />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
