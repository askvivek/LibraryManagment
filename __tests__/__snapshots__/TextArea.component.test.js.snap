// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Some component renders correctly 1`] = `
<TextInput
  allowFontScaling={true}
  multiline={true}
  onChangeText={[Function]}
  rejectResponderTermination={true}
  style={
    Array [
      Object {
        "borderColor": "gray",
        "borderWidth": 1,
        "height": 100,
        "width": 200,
      },
      Object {
        "textAlignVertical": "top",
      },
      undefined,
    ]
  }
  underlineColorAndroid="transparent"
  value=""
/>
`;
